assets/sass/config/_variables.scss

```diff
@@ -277,6 +277,8 @@ $bp-wpAdminBarTablet: $width-wpAdminBarTablet + px;
 $bp-desktop: $width-desktop + px;
 $bp-xlarge: $width-xlarge + px;
 $bp-xxlarge: $width-xxlarge + px;
+$minWidth-tablet: $width-tablet + 1 + px;
+$minWidth-desktop: $width-desktop + 1 + px;
```

Comment:
I think these should follow the convention of using `bp-` as a prefix. What do you think?

Also, we already have `$bp-nonMobile` that emits the same value as `$minWidth-tablet`. Should we probably re-use that? We may introduce `$bp-nonTablet` replacing `$minWidth-desktop`. Let me know what you think.

Reply:
I'm not opposed to the idea, but `bp-nonMobile` and `bp-nonTablet` doesn't really sound all that intuitive to me, considering we already have `bp-mobile/tablet` and those are confusing with being inclusive/exclusive.

Although I think the `minWidth-tablet/desktop` is more intuitive and, to reason about and consistent with the JS hook implementation, I'll apply your suggestion for now to make it aligned with prior art, and hopefully file a general enhancement issue to make the whole thing more intuitive  and clear in due course. 

Cheers.

Reply:
I understand and agree. I will also appreciate a follow-up issue.